src/routes/transactions/transactions.service.ts

```diff
@@ -218,6 +224,72 @@ export class TransactionsService {
     };
   }
 
+  private adjustTransactionHistoryPage(
+    limit?: number,
+    offset?: number,
```

Comment:
This should not be `undefined`

Reply:
Done 3ed0214c5e8c5f8593e35c3295694f62ae20da5e